src/main/java/redis/clients/jedis/BinaryShardedJedis.java

```diff
@@ -19,6 +21,8 @@
 public class BinaryShardedJedis extends Sharded<Jedis, JedisShardInfo> implements
     BinaryJedisCommands {
 
+  private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
Loggers should be `static` otherwise they could introduce performance issues. (Applies in a few other places too)